vm/systemSmartContracts/auction.go

```diff
@@ -810,9 +807,13 @@ func (s *stakingAuctionSC) unStake(args *vmcommon.ContractCallInput) vmcommon.Re
 		}
 
 		vmOutput, err := s.executeOnStakingSC([]byte("unStake@" + hex.EncodeToString(blsKey) + "@" + hex.EncodeToString(registrationData.RewardAddress)))
-		isError := err != nil || vmOutput.ReturnCode != vmcommon.Ok
-		if isError {
-			log.LogIfError(err)
+		if err != nil {
+			s.eei.AddReturnMessage(fmt.Sprintf("cannot do unStake for key %s, error %s", hex.EncodeToString(blsKey), err.Error()))
```

Comment:
"cannot do unStake for key %s: %s" is enough

Reply:
done.